code/game/objects/items/weapons/tools.dm

```diff
@@ -46,14 +47,39 @@
 	icon_state = "wrench_brass"
 	toolspeed = 2
 
+/obj/item/weapon/wrench/power
+	name = "Hand Drill"
+	desc ="A simple powered drill with a bolt bit"
+	icon_state = "drill_bolt"
+	item_state = "drill"
+	usesound = 'sound/items/drill_use.ogg'
+	materials = list(MAT_METAL=150,MAT_SILVER=50,MAT_TITANIUM=25)
+	origin_tech = "materials=2;engineering=6" //done for balance reasons, making them high value for research, but harder to get
+	force = 8 //might or might not be too high, subject to change
+	throwforce = 8
+	attack_verb = list("drilled", "screwed", "jabbed")
+	toolspeed = 4
+
+/obj/item/weapon/wrench/power/attack_self(mob/user)
+	playsound(get_turf(user),"sound/items/change_drill.ogg",50,1)
+	var/obj/item/weapon/wirecutters/power/s_drill = new /obj/item/weapon/screwdriver/power
+	user << "<span class='notice'>You attach the screw driver bit to the [src.name].</span>"

```

Comment:
Yeap... i knew there was going to be a spelling error that escaped my proofreading.